docs/static/install.sh

```diff
@@ -56,10 +56,11 @@ is_command() {
 # example: check_sha256 "${TMP_DIR}" checksums.txt
 validate_checksums_file() {
   cd "$1"
+  grep "$FILENAME" "$2" > checksum.txt
```

Comment:
What does this do exactly? What was the problem with alpine?

Reply:
The problem is that `sha256sum` does not recognize the following options on Alpine: `--ignore-missing`, `--quiet `. More info in the #587 issue.

Reply:
Added more context in the PR

Reply:
Thanks for the context :)